skabase/SKALogger/SKALogger/__init__.py

```diff
@@ -4,6 +4,8 @@
 #
 #
 #
+# Distributed under the terms of the none license.
+# See LICENSE.txt for more info.
```

Comment:
These lines should be removed.

Reply:
This is outdated change. It's not there in recent code.